v23.1/full-text-search.md

```diff
@@ -0,0 +1,472 @@
+
+    ~~~ sql
+    SELECT opener, response, ts_rank(joke, query) AS rank
+    FROM dadjokes, to_tsvector(opener || response) joke, to_tsquery('tree') query
+    WHERE query @@ joke
+    ORDER BY rank DESC
+    LIMIT 10;
+    ~~~
+
+    ~~~
+                     opener                |   response   |    rank
+    ---------------------------------------+--------------+--------------
+      What kind of tree fits in your hand? | A palm tree. | 0.075990885
+      How do trees get on the internet?    | They log on. |  0.06079271
+    (2 rows)
+    ~~~
+
+    The frequency of the `tree` lexeme in each row determines the difference in the rankings.
+
+1. Search the table for the query `calling`, and rank the results:
+    
+    ~~~ sql
+    SELECT opener, response, ts_rank(joke, query) AS rank
+    FROM dadjokes, to_tsvector(opener || response) joke, to_tsquery('calling') query
+    WHERE query @@ joke
+    ORDER BY rank DESC
+    LIMIT 10;
+    ~~~
+
+    ~~~
+                          opener                      |     response     |    rank
+    --------------------------------------------------+------------------+-------------
+      What would a bathroom for fancy cats be called? | The glitter box. | 0.06079271
+      What do you call a pony with a sore throat?     | A little horse.  | 0.06079271
+    (2 rows)
+    ~~~
+
+    Unlike pattern matching with `LIKE` and `ILIKE`, a full-text search for `calling` produced matches. This is because [`to_tsvector()`](#process-a-document) and [`to_tsquery()`](#form-a-query) each normalized derivatives of the word "call" in their respective inputs to the lexeme `call`, using the default `english` [text search configuration](#text-search-configuration).
+
+1. Use [`plainto_tsquery()`](#form-a-query) to convert text input to a search query:
+
+    ~~~ sql
+    SELECT opener, response, ts_rank(joke, query) AS rank
+    FROM dadjokes, to_tsvector(opener || response) joke, plainto_tsquery('no more joking, dad') query
+    WHERE query @@ joke
+    ORDER BY rank DESC
+    LIMIT 10;
+    ~~~
+
+    ~~~
+                    opener                |         response          |    rank
+    --------------------------------------+---------------------------+-------------
+      When does a joke become a dad joke? | When it becomes apparent. | 0.18681315
+    (1 row)
+    ~~~
+
+1. Alternatively, use [`phraseto_tsquery()`](#form-a-query) to search for matching text phrases (in this example, "joke dad"):
+
+    ~~~ sql
+    SELECT opener, response, ts_rank(joke, query) AS rank
+    FROM dadjokes, to_tsvector(opener || response) joke, phraseto_tsquery('no more joking, dad') query
+    WHERE query @@ joke
+    ORDER BY rank DESC
+    LIMIT 10;
+    ~~~
+
+    ~~~
+      opener | response | rank
+    ---------+----------+-------
+    (0 rows)
+    ~~~
+
+## Unsupported features
+
+Some PostgreSQL syntax and features are currently unsupported. These include, but are not limited to:
```

Comment:
Here it seems like you could remove "currently" since you have the tracking issue following, which clues users into the fact that this may be in progress.